spec/features/etd_creation_spec.rb

```diff
@@ -207,12 +207,13 @@
     Timeout.timeout(Settings.timeouts.workflow) do
       loop do
         visit "#{Settings.argo_url}/view/#{prefixed_druid}"
-        break if page.has_text?("Embargoed until #{embargo_date.to_formatted_s(:long)}", wait: 1)
+        break if page.has_text?("Embargoed until #{embargo_date.to_formatted_s(:long)}",
+                                wait: Settings.timeouts.etd.embargo_poll_interval)
```

Comment:
i made the new config value equal the old hardcoded value of 1 second.  for me, when this was 1 second, i noticed when watching the test browser that it just kept refreshing a duo URL, presumably a URL that stanford webauth bounced me to before letting me into argo, to see whether i needed another 2FA prompt.

when i upped this to 10 seconds for myself locally, the test no longer got stuck here when i ran it.

Reply:
i prefer what #517 does to deal with this problem